### DISCUSSION TITLE
X/Y Axis moving too fast at 100% - Safe print speed settings?

### DISCUSSION THREAD

**Discussion by MoltenEQ** [1coderookie/KobraGoNeoInsights#25] [Q&A] [answered]
Hi!

I noticed recently that my X and Y stepper motors move really fast while printing (and make a whiny sound while doing so) , and decided to manually dial down the print speed from 100% to 80%. I don't recall why or when this started happening, I just noticed that it was moving really fast, shaking and making high pitched sounds during a print.

I'm using a Kobra Neo, with a modded firmware (I think [jokubasver](https://github.com/jokubasver)'s). I've set up a Raspberry Pi 2B as an Octoprint server (with Octopi) and I'm running Cura as my slicer, using [sclebo05](https://github.com/sclebo05) profiles as a basis.
I also use the Cura plugin that lets me to export to Octprint right away.

My Axis speed settings are: 
- X and Y: 6000 mm/min,
- Z: 200 mm/min
- E: 300 mm/min

I can usually print a ~38g PLA part in about 3 hours. Is this what I should expect?

**Comment by 1coderookie** [comment, marked as answer]
So that would be 
- x/y = 100mm/s
- z = 3.3mm/s
- e = 5mm/s  

speed then, as your settings are in minutes and I just divided it by 60 to get the speed per second - right?
So, talking about x&y it should be fine with a well-trammed machine (edit: IF the type of filament is fine for printing at 100mm/s - TPU for example definitely isn't, and for PETG I personally usually don't go faster than 80mm/s).   

BUT: what's your setting in Cura for TRAVEL MOVEMENT speed? That would be the setting which determines the movement speed *betwee it's actually extruding*, like when one layer or line is done and it has to move to the starting point of the next layer/line. If that one is set too high, then you'd experience such a behaviour you described (if I understood your problem right.. ;) ).  

Can you upload screenshots of that speed setting section at Cura?

**Reply by 1coderookie** [reply]
Forgot: dunno about the weight of your print after 3hrs tbh, as that surely differs with the amoung of infill etc..

**Reply by MoltenEQ** [reply]
Okay, it was the travel speed... Was set to 150mm/s (The normal PLA profiles use 125mm/s)
But I think I found put why it was the was it is, I was using the "fast" profile.
Dumb mistake :D
![image](https://github.com/1coderookie/KobraGoNeoInsights/assets/78501629/59821aa3-b1be-4da1-aba6-5d9eb74dd8ce)

**Reply by MoltenEQ** [reply]
Ohh, also, while I'm here, is it okay to leave Z hop on? I had a couple instances where the print was knocked off the build plate.

**Reply by 1coderookie** [reply]
I personally have z hop disabled as it's said to cause (more) stringing. If your prints get knocked off, then I'd rather try to find out what exactly happend and why it happend. Like was there a cooling problem and therefore a certain edge curled up which then got hit by the nozzle? For example..
